lutris/gui/views/base.py

```diff
@@ -3,15 +3,15 @@
 from lutris.database.games import get_game_for_service
 from lutris.database.services import ServiceGameCollection
 from lutris.game import Game
-from lutris.game_actions import get_game_actions
+from lutris.game_actions import get_game_actions, get_multiple_game_actions
 from lutris.gui.views import COL_ID
 from lutris.gui.widgets.contextual_menu import ContextualMenu
 
 
 class GameView:
     # pylint: disable=no-member
     __gsignals__ = {
-        "game-selected": (GObject.SIGNAL_RUN_FIRST, None, (Gtk.TreeIter,)),
+        "game-selected": (GObject.SIGNAL_RUN_FIRST, None, (object,)),
```

Comment:
I think this should be list, not object - if that will work with GTK signals anyway.

Reply:
Again, the list view of games return a treeiter, not a list. Also, llist won't work with GTK signals it seems.

Reply:
It needs to be consistent. I think you used a list sometimes, so I guess you can pass a list even if you say object here.

Reply:
Will fix with the above issue for consistancy